client/app/components/packages/landuse-form/proposed-action-editor.hbs

```diff
@@ -15,9 +20,163 @@
       </span>
     </Ui::Legend>
 
+    <Ui::Question
+      class="no-margin-top"
+      @required={{true}}
+      data-test-dcpapplicantispublicagencyactions-radio-group
```

Comment:
Actually no need to add the test selector here. The beauty of the Field component is that it generates these test selectors for us